internal/common/test_configuration.go

```diff
@@ -126,6 +136,52 @@ var TestDomainNameResolutionSuccess = &models.DomainResolutionResponse{
 
 var TestDefaultRouteConfiguration = []*models.Route{{Family: FamilyIPv4, Interface: "eth0", Gateway: "192.168.1.1", Destination: "0.0.0.0"}}
 
+var TestIPv4Networking = TestNetworking{
+	ClusterNetworkCidr:       "1.3.0.0/16",
+	ClusterNetworkHostPrefix: 24,
+	ServiceNetworkCidr:       "1.2.5.0/24",
+	MachineNetworkCidr:       "1.2.3.0/24",
+	APIVip:                   "1.2.3.5",
+	IngressVip:               "1.2.3.6",
+}
```

Comment:
While you're in here changing stuff, can we maybe use one of the ip ranges specifically reserved for documentation and tests in https://en.wikipedia.org/wiki/Reserved_IP_addresses?

Reply:
This PR just making a common one. I didn't want to make big changes to the logic.
I opened a ticket for that - https://issues.redhat.com/browse/MGMT-7501

Reply:
Fair enough